subtests/docker_cli/kill/kill_utils.py

```diff
@@ -129,7 +129,8 @@ def initialize(self):
         else:
             self._init_container_normal(name)
 
-        time.sleep(self.config['wait_start'])
+        cmd = self.sub_stuff['container_cmd']
+        cmd.wait_for_ready(timeout=self.config['wait_start'])
```

Comment:
Good catch! Thank you. I hate those duplicates but don't know how to fix them. Push coming soon.